packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.base.tsx

```diff
@@ -1,15 +1,22 @@
 import * as React from 'react';
-import { IContextualMenuProps, IContextualMenuItem, ContextualMenuItemType } from './ContextualMenu.types';
+import {
+  IContextualMenuProps,
+  IContextualMenuItem,
+  ContextualMenuItemType,
+  IContextualMenuStyleProps,
+  IContextualMenuStyles
+} from './ContextualMenu.types';
 import { DirectionalHint } from '../../common/DirectionalHint';
 import { FocusZone, FocusZoneDirection, IFocusZoneProps, FocusZoneTabbableElements } from '../../FocusZone';
 import {
   IMenuItemClassNames,
-  IContextualMenuClassNames,
+  // IContextualMenuClassNames,
```

Comment:
Could you add todo notes for where code is commented out? Just to make it clear for someone coming in later that these are being left for reference because it's only been partially converted.

Reply:
cleaned up comments